fullstack_python/task_2.py

```diff
@@ -0,0 +1,48 @@
+import os
+
+file_path = './text.py'

```

Comment:
```suggestion
FILE_PATH = './text.py'

```

This can be regarded as a constant